rcgen/src/lib.rs

```diff
@@ -1602,83 +1643,6 @@ fn write_general_subtrees(writer: DERWriter, tag: u64, general_subtrees: &[Gener
 }
 
 impl Certificate {
-	/// Generates a new self-signed certificate from the given parameters.
-	///
-	/// The returned [`Certificate`] may be serialized using [`Certificate::der`] and
-	/// [`Certificate::pem`].
-	pub fn generate_self_signed(
```

Comment:
Is the next release planned to be a breaking one? If no, maybe we could keep these functions around but deprecate them.